.github/workflows/tests.yml

```diff
@@ -28,15 +28,21 @@ jobs:
 
       - name: Install test dependencies
         if: runner.os == 'macos'
-        run: brew install coreutils fish
+        run: brew install coreutils fish elvish
 
       - name: Install test dependencies
         if: runner.os == 'linux'
         run: |
-          PPA="ppa:fish-shell/nightly-master"
-          sudo add-apt-repository -y "$PPA"
+          sudo add-apt-repository -y ppa:fish-shell/nightly-master
+          sudo add-apt-repository -y ppa:longsleep/golang-backports
           sudo apt-get update
-          sudo apt-get -y install fish
+          sudo apt-get -y install fish golang-go
+
+          # Build elvish and add to path
+          git clone --depth 1 --single-branch https://github.com/elves/elvish
+          cd elvish
+          make get
```

Comment:
Is this building elvish from source? Or is this just downloading a pre-built binary?

Reply:
Building from source - didn't take too long locally, but looks like there is a linux binary available, so using that instead. Changed in 25382798d9b2d65a3867f548732a5f625f195d85